### ISSUE TITLE
Verify cluster can be updated via CloudFormation

### ISSUE THREAD

**Issue by sopel** [cityindex-attic/logsearch#119]
This details #115 and depends on #116 (and probably #39) - it should be possible to update the cluster with a single command by means of updating the stack via the existing CFN template, e.g. to adjust instance sizes or replace unhealthy instances.
- an instance can die at any time in principle, and ideally only that instance would need to be replaced rather than the entire cluster
- this is definitely best handled by [Auto Scaling](http://aws.amazon.com/autoscaling/) rather than manually triggering similar actions, thus it's finally time to tackle #39
  - once this is working, replacing unhealthy instances should work automatically even in many circumstances
  - spot instance usage is out of scope still, but would be trivial to add thereafter
  - likewise, actual scaling is out of scope still (i.e. the number of instances will be fixed), but is technically implied already, see #54 for details
